src/test/java/io/opentelemetry/OverheadTests.java

```diff
@@ -56,25 +55,48 @@ static void tearDown() {
     collector.close();
   }
 
-  @Disabled
-  @TestFactory
-  Stream<DynamicTest> runAllTestConfigurations() {
-    return Configs.all().map(config ->
-        dynamicTest(config.getName(), () -> runTestConfig(config))
-    );
+  @Test
```

Comment:
I think we probably want to delete this entirely soon, but for now we should definitely keep it disabled.

```suggestion
  @Disabled
  @Test
```